UPGRADE-6.4.md

```diff
@@ -30,6 +30,12 @@ Form
  * Deprecate using `DateTime` or `DateTimeImmutable` model data with a different timezone than configured with the
    `model_timezone` option in `DateType`, `DateTimeType`, and `TimeType`
 
+FrameworkBundle
+---------------
+
+ * Deprecate the integration of Doctrine annotations, either uninstall the `doctrine/annotations` package or disable
```

Comment:
isnt also good to document here that the way to go is native attributes?
it may seem obvious but just asking :)

Reply:
That's certainly what we should do when we deprecate Doctrine annotations for components (Routing and friends).

Reply:
Will be fixed in #49358, #50982, #50983.